x/distribution/client/cli/tx.go

```diff
@@ -192,7 +193,7 @@ $ %s tx set-withdraw-addr cosmos1gghjut3ccd8ay0zduzj64hwre2fxs9ld75ru9p --from m
 }
 
 // GetCmdSubmitProposal implements the command to submit a community-pool-spend proposal
-func GetCmdSubmitProposal(cdc *codec.Codec) *cobra.Command {
+func GetCmdSubmitProposal() *cobra.Command {
```

Comment:
So this PR moves us from having a top level codec to having all module level codecs???

Reply:
Many places where only the module level codec is required, the module level codec is used.